### ISSUE TITLE
Any plan for Rubygems?

### ISSUE THREAD

**Issue by BerkhanBerkdemir** [siimple/siimple#35]
> This issue will be a **feature** request

I am using Ruby all the time. When I add Siimple, it makes hard to follow versions. I though maybe Rubygems will be good for that situation?

Thank you

**Comment by BerkhanBerkdemir** [comment]
Here is the Gem - https://github.com/BerkhanBerkdemir/siimple-rubygem
Right now, I just tested with Rails.

When I run Rails, I am getting this error. However, it doesn't effect to much, so I can work with siimple.
```
DEPRECATION WARNING on line 70 of /home/berkhan/Projects/lorem/vendor/bundle/ruby/2.5.0/bundler/gems/siimple-rubygem-14a1d360ad07/assets/stylesheets/layout/_breadcrumb.scss:
#{} interpolation near operators will be simplified in a future version of Sass.
To preserve the current behavior, use quotes:

  unquote("- #{$siimple-breadcrumb-height / 2}")

You can use the sass-convert command to automatically fix most cases.
```
Do you have any idea?

**Comment by jmjuanes** [comment]
Hello @BerkhanBerkdemir 

>I am using Ruby all the time. When I add Siimple, it makes hard to follow versions. I though maybe Rubygems will be good for that situation? 

I have no problem to publish `siimple` (and also `siimple-colors`) in RubyGems. But the first version to be published in RubyGems will be the `v3.1.0-rc`, not the `v3.0.0`.

> When I run Rails, I am getting this error. However, it doesn't effect to much, so I can work with siimple.

Can you tell me which version of SASS are you using, please?

**Comment by BerkhanBerkdemir** [comment]
I just published but it was pre-release. I'm waiting the stable release of siimple.

**Comment by jmjuanes** [comment]
> I just published but it was pre-release. I'm waiting the stable release of siimple.

But having the gem in a separated repository is not the best solution. I think that the gem should be in the siimple repository (like bootstrap), because each new release of siimple will be automatically published in RubyGems...

**Comment by BerkhanBerkdemir** [comment]
Here it is their Gem repository https://github.com/twbs/bootstrap-rubygem

They have an Update task for main repository updates. That's why, they can keep the Gem separate from the main repository. If we can add something that trigger the Gem application, we will have auto-update system. What do you think?

**Comment by jmjuanes** [comment]
> If we can add something that trigger the Gem application, we will have auto-update system.

If it can be triggered automatically, for me perfect! :smile:

**Comment by BerkhanBerkdemir** [comment]
How will we trigger that? By hand or branch commit? If it's branch commit, will it be `master` branch?

**Comment by jmjuanes** [comment]
> How will we trigger that? By hand or branch commit? If it's branch commit, will it be master branch?

Is there any possibility of doing that with tags? If not, I usually use the `master` branch for stable releases, so we can do that with branch commit. What do you think?

> Do I need to use siimple-colors in that repository for dependency because in that line I added but I don't know why.

The new version does not have `siimple-colors` as a dependency. I think that you may wait to the release candidate.

**Comment by BerkhanBerkdemir** [comment]
* Actually I was doing like this but I wanted to ask again.
* OK. I'll wait.

**Comment by jmjuanes** [comment]
@BerkhanBerkdemir Do you want to be part of the siimple organization? I think that you should be part of the organization, because you are contributing with good ideas and improvements.

**Comment by BerkhanBerkdemir** [comment]
Yes, I do, @jmjuanes :smiley:.

**Comment by BerkhanBerkdemir** [comment]
This issue will open after the stable release. Are you OK with that, @jmjuanes?

**Comment by jmjuanes** [comment]
Perfect :ok_hand:

**Comment by jmjuanes** [comment]
Hello @BerkhanBerkdemir 

The version `v3.1.0` is ready and will be published today or tomorrow. I need some information about your gem:
- Installation instructions? To be added in the documentation and in the README.
- Badge for the README?

PS: I have added your repository to the README, in the **Related projects** section :smile:

**Comment by BerkhanBerkdemir** [comment]
Good to hear that from you, @jmjuanes :confetti_ball: :confetti_ball:   
I'm going to write Travis CI stuff but manually it works like charm :rofl:.

> Installation instructions? To be added in the documentation and in the README

I have some [instruction](https://github.com/BerkhanBerkdemir/siimple-rubygem/blob/master/README.md#installation) about Rails applications.

> Badge for the README?

Like *PRs welcome*?

**Comment by jmjuanes** [comment]
> I have some instruction about Rails applications. 

Ok, so we can add the installation instructions for gem in the documentation website, and also add a link to your repository. Do you agree?

> Like PRs welcome?

Yes, the badge with the gem version. But for that the gem should be published in www.rubygems.org. Do you have plans to publish it here?

> Also before move on 3.1.0 let's check some SCSS errors. What you think?

Mmm yes, you are right.

**Comment by BerkhanBerkdemir** [comment]
> Ok, so we can add the installation instructions for gem in the documentation website, and also add a link to your repository. Do you agree?

Think about it just finished :+1:

> Yes, the badge with the gem version. But for that the gem should be published in www.rubygems.org. Do you have plans to publish it here?

I have plan to do

**Comment by jmjuanes** [comment]
All errors and warnings have been solved: 

![image](https://user-images.githubusercontent.com/5751201/41808848-38789560-76e4-11e8-93c7-b3be32609413.png)

I am using the same configuration file that you suggested in  https://github.com/siimple/siimple/issues/39#issuecomment-394020331, but with some changes (indentation has been changed to 4 spaces and double quotes instead of single quotes): https://github.com/siimple/siimple/blob/develop/.sass-lint.yml

**Comment by BerkhanBerkdemir** [comment]
Sounds good to me :+1:, @jmjuanes. SCSS doesn't have clear answer for styling, so you can use your own. scss-lint helps you to write more reliable code.

**Comment by jmjuanes** [comment]
The `v3.1.0` has been published. Now you can publish your gem with the new version :smile:

**Comment by BerkhanBerkdemir** [comment]
Wow, it was quick, @jmjuanes. Yesterday's morning I'll look at it.
